docs/userguide/main.md

```diff
@@ -1432,3 +1432,57 @@ spec:
 ```
 
 Note: the `.spec.name` is immutable, meaning that it cannot be changed after the resources have been deployed to a Kubernetes cluster
+
+### Managing Snapshot Policies with Kubernetes Resources
+
+The OpenSearch Operator provides a custom Kubernetes resource to create, update, and manage Snapshot Lifecycle Management (SLM) policies using Kubernetes manifests. This makes it possible to declaratively define and control snapshot policies alongside your cluster resources.
+
+Fields in the CRD map directly to the OpenSearch snapshot policy structure, allowing seamless integration. Policies are not modified if they already exist in OpenSearch. You can define a new policy using the following example:
+
+```
+apiVersion: opensearch.opster.io/v1
+kind: OpensearchSnapshotPolicy
+metadata:
+  name: sample-policy
+  namespace: default
+spec:
+  policyName: sample-policy
+  enabled: true
+  description: Sample policy
+  opensearchCluster:
+    name: my-first-cluster
+  creation:
+    schedule:
+      cron:
+        expression: "0 0 * * *"
+        timezone: "UTC"
+    timeLimit: "1h"
+  deletion:
+    schedule:
+      cron:
+        expression: "0 1 * * *"
+        timezone: "UTC"
+    timeLimit: "30m"
+    deleteCondition:
+      maxAge: "7d"
+      maxCount: 10
+      minCount: 3
+  snapshotConfig:
+    repository: sample-repository
+    indices: "*"
+    includeGlobalState: true
+    ignoreUnavailable: false
+    partial: false
+    dateFormat: "yyyy-MM-dd-HH-mm"
+    dateFormatTimezone: "UTC"
+    metadata:
+      createdBy: "sample-operator"
+```
+
+#### Note:
+
+- The OpensearchSnapshotPolicy must be created in the same namespace as the OpenSearch cluster it targets.
+
+- `policyName` is an optional field, and if not provided `metadata.name` is used as the default.
+
+- The repository field must reference an existing snapshot repository in the OpenSearch cluster.
```

Comment:
Can you please add a link https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/docs/userguide/main.md#configuring-snapshot-repositories here which shows how to configure the snapshot repos.